chemprop/nn/loss.py

```diff
@@ -23,13 +22,12 @@
     "DirichletMixin",
     "BinaryDirichletLoss",
     "MulticlassDirichletLoss",
-    "_ThresholdMixin",
     "SIDLoss",
     "WassersteinLoss",
 ]
 
 
-class LossFunction(ABC, ReprMixin):
+class LossFunction(ABC, nn.Module):
```

Comment:
```suggestion
class LossFunction(nn.Module):
```
`Module`s are `ABC`s